includes/models/model.llms.lesson.php

```diff
@@ -142,6 +142,35 @@ public function get_available_date( $format = '' ) {
 		// Default availability is the element's post date.
 		$available = $this->get_date( 'date', 'U' );
 
+		// get the course setting first, if any.
+		$course = $this->get_course();
+		if ( $course && 'yes' === $course->get( 'lesson_drip' ) ) {
+			$course_drip_method = $course->get( 'drip_method' );
+
+			switch ( $course_drip_method ) {
+				case 'start':
+					$ignore_lessons = intval( $course->get( 'ignore_lessons' ) );
+					$course_lessons = $course->get_lessons( 'ids' );
+					$lesson_number = array_search( $this->get( 'id' ), $course_lessons ) + 1;
```

Comment:
Can't just use `$this->get( 'order' )` here because it's the order within the given section. And since sections can have a different number of lessons each, you can't easily do something like `$this->get_section()->get( 'order' ) * X + $this->get( 'order' )`